src/ansys/platform/instancemanagement/exceptions.py

```diff
@@ -0,0 +1,111 @@
+"""Exceptions raised by PyPIM."""
+
+
+import grpc
+
+
+class NotConfiguredError(RuntimeError):
+    """An attempt was made to use PyPIM without the mandatory configuration.
```

Comment:
@plule-ansys Because of libraries using summary tables, Alex and I decided to use a verb ending in an "s" for class descriptions and not ending in an "s" for functions/methods. If you prefer a verb other than "Indicates" for these classes, feel free to change!